dotCMS/src/main/java/com/dotmarketing/portlets/rules/util/RulePermissionableUtil.java

```diff
@@ -37,9 +37,12 @@ public static Permissionable findParentPermissionable(String parent) throws DotD
                 if(iden.getAssetType().equals("folder")){
                     permissionableParent = APILocator.getFolderAPI().find(parent,systemUser,false);
                 }else{
-					Contentlet contentlet = APILocator.getContentletAPI().findContentletByIdentifier(parent, false,
-							APILocator.getLanguageAPI().getDefaultLanguage().getId(), systemUser, false);
-					if (contentlet.isHost()) {
+					//Contentlet contentlet = APILocator.getContentletAPI().findContentletByIdentifier(parent, false,
+					//		APILocator.getLanguageAPI().getDefaultLanguage().getId(), systemUser, false);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)